src/domains/ecommerce-common/03_cosmosdb.tf

```diff
@@ -143,6 +143,15 @@ locals {
       ]
       shard_key = "_id"
     },
+    {
+      name = "user-stats"
+      indexes = [{
+        keys   = ["_id"]
+        unique = true
+        }
+      ]
+      shard_key = "_id"
```

Comment:
what is this id? does it reflect these characteristics?
https://www.mongodb.com/docs/manual/core/sharding-choose-a-shard-key/

Reply:
this id is the userId as returned by PDV, that is an UUID associated to the user fiscal code tokenization